rialto_airflow/harvest/openalex.py

```diff
@@ -92,7 +91,7 @@ def publications_from_dois(dois: list):
         # TODO: do we need this to stay within 100,000 requests / day API quota?
         time.sleep(1)
 
-        doi_list = quote("|".join([doi for doi in doi_batch]))
+        doi_list = "|".join([doi for doi in doi_batch])
```

Comment:
The problem with `&` appearing in the DOI has now been fixed in pyalex:

https://github.com/J535D165/pyalex/pull/42